terminus/tests/city_test.py

```diff
@@ -57,3 +62,29 @@ def test_add_intersection_at_on_different_z_order(self):
         self.city.add_road(street2)
         self.assertEquals(street1.get_nodes()[0], RoadIntersectionNode.on(0, 0))
         self.assertEquals(street2.get_nodes()[0], RoadSimpleNode.on(0, 0, 1))
+
+    def test_bounding_box_with_ground_plane(self):
+        building = Building.square(Point(35, 45), 10, 10)
+        block = Block.square(Point(-75, -45), 10)
+        road = Road.from_points([Point(60, -20), Point(80, -20)])
```

Comment:
Create a street instead of creating a Road + adding a lane. A street is a road that has a single lane at offset 0.